grid_apps/AppManager.js

```diff
@@ -134,7 +183,7 @@ class AppManager extends EventEmitter {
 
     let url = app.url || 'http://localhost:3000'
     const mainWindow = createRenderer(
-      WindowManager.getMainUrl(),
+      await getGridUiUrl(), // FIXME might be very inefficient. might load many grid-ui packages into memory!!
```

Comment:
Any ideas for this one?

Reply:
After discussion at standup: okay to punt this to next release. We expect few windows to be created, so not critical at this stage.